src/main/kotlin/no/nav/syfo/tilgang/TilgangService.kt

```diff
@@ -62,6 +62,34 @@ class TilgangService(
         return tilgang
     }
 
+    private suspend fun isKode6AccessAvslatt(token: Token, callId: String): Boolean {
+        return !graphApiClient.hasAccess(adRolle = adRoller.KODE6, token = token, callId = callId)
+    }
+
+    private suspend fun isKode7AccessAvslatt(token: Token, callId: String): Boolean {
+        return !graphApiClient.hasAccess(adRolle = adRoller.KODE7, token = token, callId = callId)
+    }
+
+    private suspend fun isAdressebeskyttelseAccessGodkjent(
```

Comment:
Jeg ga denne tilsvarende navn som den for skjerming, men de brukes jo kun negert (if ikke-isGodkjent). Jeg mener koden ble litt finere sist da jeg gjorde det sånn, men det er kanskje bedre å gjøre om så metodenavnene er isAvslatt? 🤔

Reply:
Synes det ser ok ut sånn du har gjort det 👍

Reply:
Hmm, eneste jeg synes var litt rart var at jeg leste `avslatt` som "turned off" i stedet for "denied" 🤔